web/templates/inbox_stream_container.hbs

```diff
@@ -0,0 +1,10 @@
+{{#each topics_dict }}
+    <div id="{{@key}}">
+        {{> inbox_row (lookup ../streams_dict @key)}}
+        <div class="topic_contianer">
+            {{#each this}}
+                {{>inbox_row this}}
+            {{/each}}
+        </div>
+    </div>
+{{/each}}
```

Comment:
What's the thinking behind this being a separate template while the `pms_dict` is just inline in the `inbox_view.hbs` template? I think I prefer inline if there's not a rerendering-related reason to do it this way, but I'm guessing there is -- I guess to have the stream headers or something?

Reply:
Yes.